### PR TITLE
fix(console): better error messages

### PR DESCRIPTION
**Pull request by eliobischof**
- Shows validation errors on all validated fields
- Translation keys are centrally defined on validators
- Removes default country Switzerland from phone input

Complements #5382 
Closes: #5038 

```[tasklist]
### Definition of Ready
- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [x] PR is linked to the corresponding user story
- [x] Acceptance criteria are met
- [x] All open todos and follow ups are defined in a new ticket and justified
- [x] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [x] No debug or dead code
- [x] Critical parts are tested automatically
- [x] Where possible E2E tests are implemented
- [x] Documentation/examples are up-to-date
- [x] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

**Comment by eliobischof** [comment]
@peintnermax I think what you addressed in the first review was the bad practice of manually subscribing to observables?

The new component subscribes in the template using the async pipe.
Do you think there are other best practices violated in this PR?
Didn't find one here https://blog.brecht.io/rxjs-best-practices-in-angular/#avoiding-manual-subscribes-in-Angular

File: console/src/app/modules/form-field/field/form-field.component.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<div class="cnsl-form-field-wrapper" (click)="_control.onContainerClick && _control.onContainerClick($event)">
+  <ng-content select="cnsl-label"></ng-content>
+  <div class="cnsl-rel" #inputContainer>
+    <ng-content></ng-content>
+    <ng-content select="cnslSuffix"></ng-content>
+  </div>
+  <div *ngIf="errori18nKeys$ | async; let errs">
+    <div class="cnsl-form-field-subscript-wrapper">
```
Comment (peintnermax) [inline console/src/app/modules/form-field/field/form-field.component.html:8]: move class to the parent

File: console/src/app/modules/form-field/field/form-field.component.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <div class="cnsl-form-field-hint-wrapper" *ngIf="errs.length === 0" [@transitionMessages]="_subscriptAnimationState">
+        <ng-content select="cnsl-hint"></ng-content>
+      </div>
+      <div class="cnsl-form-field-error-wrapper" *ngIf="errs.length > 0" [@transitionMessages]="_subscriptAnimationState">
```
Comment (peintnermax) [inline console/src/app/modules/form-field/field/form-field.component.html:12]: ensure that other error directives are included as well

File: console/src/app/modules/form-field/field/form-field.component.ts [hunk 3]
```diff
@@ -86,13 +94,12 @@ export class CnslFormFieldComponent extends CnslFormFieldBase implements OnDestr
     @Inject(ElementRef)
     _labelOptions: // Use `ElementRef` here so Angular has something to inject.
     any,
+    private kvPipe: KeyValuePipe,
```
Comment (peintnermax) [inline console/src/app/modules/form-field/field/form-field.component.ts:97]: remove and use native typescript

**Review by peintnermax — CHANGES_REQUESTED** [review]
please ensure custom errors can still be shown, and i18nErrors can be disabled via a input on the formfield component

File: console/src/app/modules/form-field/field/form-field.component.html (hunk 1 above)
Comment (eliobischof) [reply console/src/app/modules/form-field/field/form-field.component.html]: done

File: console/src/app/modules/form-field/field/form-field.component.html (hunk 2 above)
Comment (eliobischof) [reply console/src/app/modules/form-field/field/form-field.component.html]: done

File: console/src/app/modules/form-field/field/form-field.component.ts (hunk 3 above)
Comment (eliobischof) [reply console/src/app/modules/form-field/field/form-field.component.ts]: done

File: console/src/app/modules/form-field/field/form-field.component.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  <div class="cnsl-rel" #inputContainer>
+    <ng-content></ng-content>
+    <ng-content select="cnslSuffix"></ng-content>
+  </div>
+  <ng-container *ngIf="help$ && help$ | async as help">
+    <div [ngSwitch]="help.type" class="cnsl-form-field-subscript-wrapper">
+      <div *ngSwitchCase="'hints'" class="cnsl-form-field-hint-wrapper" [@transitionMessages]="_subscriptAnimationState">
+        <ng-content select="cnsl-hint"></ng-content>
+      </div>
+      <div class="cnsl-form-field-error-wrapper" [@transitionMessages]="_subscriptAnimationState"></div>
```
Comment (peintnermax) [inline console/src/app/modules/form-field/field/form-field.component.html:12]: either delete this line or add `<ng-content select="cnsl-error"></ng-content>` here, or add the cnsl-form-field-error-wrapper class to the ngSwitchCase below

**Review by peintnermax — CHANGES_REQUESTED** [review]
LGTM but the formfield doesn't show the error message onblur. Once the onBlur event is fired, the error messages should be shown not after a change event.

File: console/src/app/modules/form-field/field/form-field.component.html (hunk 4 above)
Comment (eliobischof) [reply console/src/app/modules/form-field/field/form-field.component.html]: done

**Review by peintnermax — APPROVED** [review]

**Pull request merged**